app/controllers/events_controller.rb

```diff
@@ -10,6 +10,8 @@ def show
 
   def index
     @events = Event.all.includes(:creator)
+    @future_events = @events.active
+    @past_events = @events.expired
```

Comment:
do you need to `includes(:creator)` on these too? That could be moved to another scope as well, something like
```ruby
scope :creator_included, -> { includes(:creator) }
```

```suggestion
    @future_events = @events.active.creator_included
    @past_events = @events.expired.creator_included
```

Reply:
`@event` already includes :creator